Esri/arcgis-maps-sdk-kotlin-samples
# update new module script to use geo-compose toolkit component

## Description
PR to update the new module script to spin up new samples using the `geo-compose` toolkit module by default.

## Links and Data

Sample Epic: `runtime/kotlin/issues/3370`

## What To Review

-  Review script files

## How to Test

- Test if the JAR creates a new sample as expected.
- To run the script and create a new sample:
`java -jar tools/NewModuleScript.jar`
  The .jar file will prompt you to type in the name of the new sample.
